connector-splunk/src/main/java/com/redhat/cloud/notifications/connector/splunk/SplunkRouteBuilder.java

```diff
@@ -0,0 +1,61 @@
+package com.redhat.cloud.notifications.connector.splunk;
+
+import com.redhat.cloud.notifications.connector.ConnectorConfig;
+import com.redhat.cloud.notifications.connector.EngineToConnectorRouteBuilder;
+import org.apache.http.conn.ssl.NoopHostnameVerifier;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import static com.redhat.cloud.notifications.connector.ConnectorToEngineRouteBuilder.SUCCESS;
+import static com.redhat.cloud.notifications.connector.ExchangeProperty.ID;
+import static com.redhat.cloud.notifications.connector.ExchangeProperty.ORG_ID;
+import static com.redhat.cloud.notifications.connector.ExchangeProperty.TARGET_URL;
+import static com.redhat.cloud.notifications.connector.splunk.ExchangeProperty.ACCOUNT_ID;
+import static com.redhat.cloud.notifications.connector.splunk.ExchangeProperty.AUTHENTICATION_TOKEN;
+import static com.redhat.cloud.notifications.connector.splunk.ExchangeProperty.TARGET_URL_NO_SCHEME;
+import static com.redhat.cloud.notifications.connector.splunk.ExchangeProperty.TRUST_ALL;
+import static com.redhat.cloud.notifications.connector.splunk.SplunkTrustAllManager.getSslContextParameters;
+import static org.apache.camel.LoggingLevel.INFO;
+import static org.apache.camel.builder.endpoint.dsl.HttpEndpointBuilderFactory.HttpEndpointBuilder;
+
+@ApplicationScoped
+public class SplunkRouteBuilder extends EngineToConnectorRouteBuilder {
```

Comment:
This class is a complete rewriting of [SplunkIntegration.java](https://github.com/RedHatInsights/eventing-integrations/blob/3faa2fb8c58002ce5473166ade7181ece4fb14ed/splunk-quarkus/src/main/java/com/redhat/console/integrations/splunk/SplunkIntegration.java)